api/v1alpha1/nonadminbackup_types.go

```diff
@@ -88,6 +88,12 @@ type NonAdminBackupStatus struct {
 	// phase is a simple one high-level summary of the lifecycle of an NonAdminBackup.
 	Phase NonAdminPhase `json:"phase,omitempty"`
 
+	// logsPath is path to backup logs in the storage location
```

Comment:
Should we make struct here ?
like `OperationInfo` / `DebugInfo` / `InfoPath` / DebugFiles` os something like that, and then add these paths under that struct ?'

Reply:
depends how big you think this is gonna get

Reply:
I think if there are to be that many, we just provide one base url, and then user/cli can populate the rest. its common enough that I think specifying each is wasteful of etcd.

Reply:
If we do add a struct, do we make it inline? such that it appears the same level as Phase.. or is the intent to make it a nested status field?

Reply:
In that case, we should just provide the base url (no separate url for logs and resources as well) and document the dir structure for users, wdyt ?

Reply:
That's fine by me. I do wanna make sure everyone is heard tho.. perhaps next week's scrum?

Reply:
I agree, works for me.

Reply:
We have decided on baseURL + file paths.. simple enough.

In that case, a struct would allow for sub status fields allowing for filtering using jsonpath